Add `tubingen_apg` profile (#487)
nf-core/configs: docs/tubingen_apg.md

# nf-core/configs: University of Tuebingen APG group Configuration

All nf-core pipelines have been successfully configured for use on the APG group's clusters at the [University of Tuebingen](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/geowissenschaften/arbeitsgruppen/urgeschichte-naturwissenschaftliche-archaeologie/ina/archaeo-and-palaeogenetik/).

To use, run the pipeline with `-profile tubingen_apg`. This will download and launch the [`tubingen_apg.config`](../conf/tubingen_apg.config) which has been pre-configured with a setup suitable for the APG cluster. The number of parallel jobs that run is currently limited to 8.

This configuration will automatically choose the correct SLURM queue (short,medium,long) depending on the time and memory required by each process.

Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:

```
/opt/container_tmp/apptainer/nextflow
```

> > NB: Nextflow will need to submit the jobs via SLURM to the clusters and as such the commands above will have to be executed on one of the head nodes. If in doubt contact IT.

Reviewer: ```suggestion
> NB: The commands to run a pipeline will have to be executed on one of the head nodes, so Nextflow can submit the jobs to the clusters via SLURM. If in doubt, contact IT.
```

# nf-core/configs: University of Tuebingen APG group Configuration

All nf-core pipelines have been successfully configured for use on the APG group's clusters at the [University of Tuebingen](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/geowissenschaften/arbeitsgruppen/urgeschichte-naturwissenschaftliche-archaeologie/ina/archaeo-and-palaeogenetik/).

To use, run the pipeline with `-profile tubingen_apg`. This will download and launch the [`tubingen_apg.config`](../conf/tubingen_apg.config) which has been pre-configured with a setup suitable for the APG cluster. The number of parallel jobs that run is currently limited to 8.

This configuration will automatically choose the correct SLURM queue (short,medium,long) depending on the time and memory required by each process.

> Using this profile, a docker image containing all of the required software will be downloaded, and converted to a charliecloud container before execution of the pipeline. The image will be stored in the user-specific directory:
> 
> ```
> /opt/images-live/$USER/nf-core
> ```
> 
> The following directories are bound to the container on execution:
> - `/nfs/geo`
> - `/home`
> - `/local_scratch`
> - `/opt/reference`
> - `/opt/resources`

Reviewer 2: ```suggestion
Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:

```
/opt/container_tmp/apptainer/nextflow
```
```

> - `/opt/resources`

Author: ```suggestion
Using this profile, a docker image containing all of the required software will be downloaded, and converted to a singularity container before execution of the pipeline. The image will be stored in the cache directory:
`/opt/container_tmp/apptainer/nextflow`
```
